src/services/user/OAuthServices.ts

```diff
@@ -0,0 +1,10 @@
+import { Logger } from 'src/types/Logger';
+
+export default class OAuthServices {
+    constructor(private readonly _logger: Logger) {}
+
+    public async google(user: any): Promise<any> {
+        this._logger('info', 'Register in database made with success through google');
```

Comment:
Sugestão de mensagem:
```suggestion
        this._logger('info', 'Registration in the database made with Google was successful');
```